packages/synapse-react-client/src/components/ProjectStorage/ProjectDataAvailability.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react'
+import { Box, SxProps, Tooltip, Typography } from '@mui/material'
+import { useTheme } from '@mui/material'
+import { useSynapseContext } from '../../utils'
+import { useProjectStorageUsage } from '../../synapse-queries'
+import { SYNAPSE_STORAGE_LOCATION_ID } from 'src/synapse-client'
+import HelpPopover from '../HelpPopover'
+import { calculateFriendlyFileSize } from 'src/utils/functions/calculateFriendlyFileSize'
+
+export type ProjectDataAvailabilityProps = {
+  projectId?: string
+  sx?: SxProps
+}
+const usageBarWidth = 142 //px
+
+export const ProjectDataAvailability: React.FunctionComponent<
+  ProjectDataAvailabilityProps
+> = ({ projectId, sx }) => {
+  const { accessToken } = useSynapseContext()
+  const isLoggedIn = !!accessToken
+  const { data } = useProjectStorageUsage(projectId!, {
+    enabled: !!projectId && isLoggedIn,
+  })
+
+  const projectDataUsageArray = data?.locations.filter(
+    v => parseInt(v.storageLocationId) == SYNAPSE_STORAGE_LOCATION_ID,
+  )
+  const synapseStorageUsage =
+    projectDataUsageArray?.length == 1 ? projectDataUsageArray[0] : undefined
+  if (!synapseStorageUsage) {
+    return <></>
+  }
+  const { sumFileBytes = 0, maxAllowedFileBytes = 1 } = synapseStorageUsage
+  const usageBarFilledPx = Math.min(
+    Math.round((sumFileBytes / maxAllowedFileBytes) * usageBarWidth),
+    usageBarWidth,
+  )
```

Comment:
What happens if `maxAllowedFileBytes` is 0?

Reply:
Bad things with the current code, but what should be shown in this case?  Showing a bar with a max of 0 seems confusing, so maybe we should just show nothing.